src/main/java/pl/wrapper/parking/infrastructure/inMemory/ParkingDataRepository.java

```diff
@@ -0,0 +1,16 @@
+package pl.wrapper.parking.infrastructure.inMemory;
+
+import java.util.HashMap;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParkingDataRepository extends InMemoryRepositoryImpl<Integer, DummyObject> { // <Key, value>
```

Comment:
It's good practice to name custom components, add name like "ParkingDataRepository"